Directory.Packages.props

```diff
@@ -30,6 +30,7 @@
     <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="$(AspnetVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Identity.Stores" Version="$(AspnetVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Can we use `System.Text.Json` instead and remove this dependency?

Reply:
Removed Newtonsoft.Json dependency, thanks!